creating_images/sti.adoc

```diff
@@ -17,10 +17,11 @@ output.
 
 The main advantage of using STI for building reproducible Docker images is the
 ease of use for developers. As a builder image author, you must be aware of the
-two basic requirements for the best possible STI performance: the required image
-contents and STI scripts.
+two basic requirements for the best possible STI performance: link:#build-process[the build process]
```

Comment:
I don't like how this reads... the build process is not a "requirement" it's a thing.  image contents was a requirement.

As a builder image author, you must understand two basic concepts in order for your images to provide the best possible S2I performance: